### ISSUE TITLE
'Send password' on login page doesn't work

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#12]
It just doesn't appear to do anything, no message ever received.

**Comment by Woseseltops** [comment]
Appears to be related to the fact the Apache user for Signbank has no email address. I've requested one.

Two notes:
* The email does turn up in the logs. It currently says: 

> You're receiving this email because you requested a password reset for your user account at 
> Signbank. Please go to the following page and choose a new password: 
> http://signbank.science.ru.nl/accounts/password/reset/MQ-48x-8cb1c1ba20fc4f892e62/ Thanks for 
> using our site! 
>
> The Signbank team

* About the interface: the Dutch translation incorrectly asks for a username instead of an email.

**Comment by Woseseltops** [comment]
YES! After a lot of fiddling, I finally figured out the problem: there were some Django email settings that should simply be removed. It now works. I played around with the text a little bit, and now we have a mixed Dutch/English email, because the new texts I created have no translation yet.

**Comment by ocrasborn** [comment]
Related: https://github.com/Signbank/FinSL-signbank/issues/132

**Comment by Woseseltops** [comment]
Good one, I gave Henri some details.

**Comment by ocrasborn** [comment]
Hi all, this is broken again. An error page follows.

**Comment by vanlummelhuizen** [comment]
This probably broke when moving from Spitfire to Applejack.

**Comment by Woseseltops** [comment]
I had a quick look, and I'm confused by the following things:
* It seems to be visiting the view password_reset, but it complains about password_reset_done .
* I was thinking this was related to upgrading, but apparently not, as the ASL Signbank shows the same problem (once you fix the url problem).
* This makes me think it never worked, but it apparently did last year...

**Comment by Woseseltops** [comment]
Okay, I managed to figure out that the registration module was still using functions instead of classes, which has been deprecated for some time; when I fixed that, some things started to work again. This still does not explain why it also does not work for the ASL-Signbank, which is not upgraded to the new Django yet. 

Unfortunately, the errors still pop up when:
1. The email is sent
2. You want to save your newly picked password (which prevents the whole thing from working)

I really don't understand why the error is still there, since we're now using the new system - something for next time.
